plugins/worldinterface/src/worldproxy.h

```diff
@@ -197,13 +197,31 @@ class WorldProxy:public GazeboYarpPlugins::WorldInterfaceServer
    * @return true if success, false otherwise
    */
   virtual bool attach(const std::string& id, const std::string& link_name);
+
+  /**
+    * Attach an object to a link of the robot - takes unscoped names as arguments
```

Comment:
Can you elabore a bit more about what you mean with `unscoped` in this context. Feel free to refer to functions in Gazebo API if necessary.

Reply:
Sorry, I never saw this reply. This detail about the difference between scoped/unscoped should go in the world interface doxygen documentation.